lib/commonQueries.js

```diff
@@ -20,6 +20,7 @@ export async function loadEvidences(serviceUri, { graph, type, includeUuid, sudo
           dct:description
           dct:title
           dct:source
+          sh:order
```

Comment:
I added this to _all_ listings, even if they have a `maxCount` set to 1. This isn't really needed, but this allows us to change or remove the `maxCount` without having to edit this service.

Reply:
good!